tests/.editorconfig

```diff
@@ -3,8 +3,10 @@ dotnet_diagnostic.CA1003.severity = none # Use generic event handler instances
 dotnet_diagnostic.CA1034.severity = none # Do not nest type
 dotnet_diagnostic.CA1051.severity = none # Do not declare visible instance fields
 dotnet_diagnostic.CA1062.severity = none # In externally visible method, validate parameter is non-null before using it
+dotnet_diagnostic.CA1510.severity = none # Use 'ArgumentNullException.ThrowIfNull' instead of explicitly throwing a new exception instance - remove once we drop LACKS_THROWIFNULL
 dotnet_diagnostic.CA1707.severity = none # Remove the underscores from member name
 dotnet_diagnostic.CA1822.severity = none # Member does not access instance data and can be marked as static
+dotnet_diagnostic.CA1861.severity = none # Prefer 'static readonly' fields over constant array arguments if the called method is called repeatedly and is not mutating the passed array

```

Comment:
A lot of instances of constant arrays we used one-off in tests. Didn't seem worth it to track them down. Can revisit if you like.